Runtime/Scripts/Logger.cs

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2018-2021 MobiledgeX, Inc. All rights and licenses reserved.
+ * MobiledgeX, Inc. 156 2nd Street #408, San Francisco, CA 94105
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using UnityEngine;
+
+namespace MobiledgeX
+{
+    public class Logger
+    {
+        public enum LogType { All, ErrorsAndWarnings, ErrorsOnly};
+        static MobiledgeXSettings settings = Resources.Load<MobiledgeXSettings>("MobiledgeXSettings");
+        public static void LogWarning(string message)
```

Comment:
Are we not handling LogError the same way?

Reply:
For Errors we use UnityEngine.Debug.LogError we don't want to block since it important, for others warnings and logs, we might want to prevent it from affecting any app in production.